alkos/kernel/src/kernel.cpp

```diff
@@ -14,12 +6,26 @@
 #error "AlkOS needs to be compiled with a x86_64-elf compiler"
 #endif
 
-/* Make kernel entry C compatible */
-extern "C"
-{
+#ifdef ALKOS_TEST
+#include "../test/tester.hpp"
```

Comment:
I'm 99% sure that includes shoudn't be done with a relative path.
I do believe that it is currently set up in such a way that you should be able to just do #include "init.hpp" 
not sure about the test folder. And if it isn't, it should be changed to be like that, because header import's relative to the source file location will quickly get out of hand.

Quote from google c++ style guide:
"All of a project's header files should be listed as descendants of the project's source directory without use of UNIX directory aliases . (the current directory) or .. (the parent directory)."

Reply:
fixed